### PR TITLE
Adding tests

### PR DESCRIPTION
**Pull request by sjongejan**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,5 +1,5 @@
 # Laravel-BAG
-
+[![Build Status](https://travis-ci.org/sjongejan/laravel-bag.svg?branch=master)](https://travis-ci.org/sjongejan/laravel-bag)
```
Comment (FaaPz) [inline README.md]: Awwhhh yeah! 🎉

File: src/Response/BaseResponse.php [hunk 2]
```diff
@@ -61,9 +61,9 @@ private function isSuccessful(): bool
         switch ($statusCode) {
             case 403:
             case 401:
-                throw new UnauthorizedHttpException($this->response);
+                throw new UnauthorizedHttpException('Token');
```
Comment (FaaPz) [inline src/Response/BaseResponse.php]: > Token

☝️ Is that the exception message?

File: tests/ClientTest.php [hunk 3]
```diff
@@ -0,0 +1,36 @@
...
+    private $bagClient;
+
+    protected function setUp()
+    {
+        parent::setUp();
+
+        $this->bagClient = new BagClient();
+    }
+
+    public function testSuccessfulGetAddress()
```
Comment (FaaPz) [inline tests/ClientTest.php]: Maybe `testSuccessfullyRetrievedAddress`?

File: tests/Request/CityRequestTest.php [hunk 4]
```diff
@@ -0,0 +1,30 @@
+<?php
+
+use Sjongejan\LaravelBag\Request\CityRequest;
+
+class CityRequestTest extends BagTestCase
+{
+    public function testProperlyFormatForId()
```
Comment (FaaPz) [inline tests/Request/CityRequestTest.php]: `testProperlyFormattedIdentifier` 👈

File: tests/Request/CityRequestTest.php [hunk 5]
```diff
@@ -0,0 +1,30 @@
...
+        $cityRequest = new CityRequest();
+
+        $cityRequest->setId(' 1234 ');
+        $this->assertEquals($cityRequest->getId(), '1234');
+
+        $cityRequest->setId('12 34');
+        $this->assertEquals($cityRequest->getId(), '1234');
+    }
+
+    public function testBuildRequest()
```
Comment (FaaPz) [inline tests/Request/CityRequestTest.php]: `testBuildApiRequestBy...`? 🤔

File: tests/Request/NumberingRequestTest.php [hunk 6]
```diff
@@ -0,0 +1,46 @@
+<?php
+
+use Sjongejan\LaravelBag\Request\NumberingRequest;
+
+class NumberingRequestTest extends BagTestCase
+{
+    public function testProperlyFormatForZipCode()
```
Comment (FaaPz) [inline tests/Request/NumberingRequestTest.php]: You can use a [data provider](https://phpunit.de/manual/current/en/writing-tests-for-phpunit.html#writing-tests-for-phpunit.data-providers) here.

```php
/**
 * @dataProvider dpWronglyFormattedZipCodes
 */
public function testProperlyFormatForZipCode($wronglyFormattedZipCode)
{
    $numberingRequest->setZipCode($wronglyFormattedZipCode);

    $this->assertEquals($numberingRequest->getZipCode(), '7311KZ');
}

/**
 * @return array
 */
public function dpWronglyFormattedZipCodes()
{
    return [
        [' 7311KZ '],
        ['7311 KZ'],
        ['7311kz'],
    ];
}
```

File: tests/Request/NumberingRequestTest.php [hunk 7]
```diff
@@ -0,0 +1,46 @@
...
+        $numberingRequest = new NumberingRequest();
+
+        $numberingRequest->setStreetNumber(110);
+        $this->assertEquals($numberingRequest->getStreetNumber(), 110);
+
+        $this->expectException(TypeError::class);
+        $numberingRequest->setStreetNumber('AA');
+    }
+
+    public function testBuildRequest()
```
Comment (FaaPz) [inline tests/Request/NumberingRequestTest.php]: Please use better names. 🙏

File: tests/Request/PublicSpaceRequestTest.php [hunk 8]
```diff
@@ -0,0 +1,30 @@
...
+        $publicSpaceRequest = new PublicSpaceRequest();
+
+        $publicSpaceRequest->setId(' 1234 ');
+        $this->assertEquals($publicSpaceRequest->getId(), '1234');
+
+        $publicSpaceRequest->setId('12 34');
+        $this->assertEquals($publicSpaceRequest->getId(), '1234');
+    }
+
+    public function testBuildRequest()
```
Comment (FaaPz) [inline tests/Request/PublicSpaceRequestTest.php]: Better... NAMING! 😇

File: src/Response/BaseResponse.php (hunk 2 above)
Comment (sjongejan) [reply src/Response/BaseResponse.php]: The first parameter of the UnauthorizedHttpException is a $challenge.

`Token` is the challenge.

File: README.md (hunk 1 above)
Comment (sjongejan) [reply README.md]: 👌🏻

**Pull request merged**